cohere/client.py

```diff
@@ -296,6 +319,39 @@ def chat(
                 >>>     return_prompt=True)
                 >>> print(res.text)
                 >>> print(res.prompt)
+            Generate search queries for fetching documents to use in chat:
+                >>> res = co.chat(
+                >>>     "What is the height of Mount Everest?",
+                >>>      search_queries_only=True)
```

Comment:
up to you, but this is the most niche use case, so imo probably should be at the bottom.